internal/cri/server/container_stop.go

```diff
@@ -106,9 +106,10 @@ func (c *criService) stopContainer(ctx context.Context, container containerstore
 			}
 			return c.cleanupUnknownContainer(ctx, id, container, sandboxID)
 		}
-
+		added := make(chan struct{})
+		close(added)
```

Comment:
I'm not sure I understand this. Can you elaborate what is the motivation to pass closed channel?

Reply:
I think  here  em.c.containerStore.Get can get a not null container. so I use a closed chan

Reply:
done, thank you. choose another way.